pmm-app/tests/pages/qanPage.js

```diff
@@ -52,6 +52,13 @@ module.exports = {
     tablesTabContents: "//div[@class='card-body']//pre",
     copyQueryButton: "//button[@id='copyQueryExample']",
     spinnerLocator: "//i[@class='fa fa-spinner fa-spin spinner']",
+    newQANPanelContent: '.panel-content',
+    countOfItems: '.css-ni562u',
```

Comment:
`css-ni562u` looks like a generated class name, pls see my comment on this below.